hummingbot/connector/exchange/coinzoom/coinzoom_constants.py

```diff
@@ -0,0 +1,60 @@
+# A single source of truth for constant variables related to the exchange
+class Constants:
+    """
+    API Documentation Links:
+    https://api-docs.coinzoom.com/
+    https://api-markets.coinzoom.com/
+    """
+    EXCHANGE_NAME = "coinzoom"
+    REST_URL = "https://api.coinzoom.com/api/v1/public"
+    # REST_URL = "https://api.stage.coinzoom.com/api/v1/public"
+    WS_PRIVATE_URL = "wss://api.coinzoom.com/api/v1/public/market/data/stream"
+    # WS_PRIVATE_URL = "wss://api.stage.coinzoom.com/api/v1/public/market/data/stream"
+    WS_PUBLIC_URL = "wss://api.coinzoom.com/api/v1/public/market/data/stream"
+    # WS_PUBLIC_URL = "wss://api.stage.coinzoom.com/api/v1/public/market/data/stream"
+
+    HBOT_BROKER_ID = "CZ_API_HBOT"
+
+    ENDPOINT = {
+        # Public Endpoints
+        "TICKER": "marketwatch/ticker",
+        "SYMBOL": "instruments",
+        "ORDER_BOOK": "marketwatch/orderbook/{trading_pair}/150/2",
+        "ORDER_CREATE": "orders/new",
+        "ORDER_DELETE": "orders/cancel",
+        "ORDER_STATUS": "orders/list",
+        "USER_ORDERS": "orders/list",
+        "USER_BALANCES": "ledger/list",
+    }
+
+    WS_SUB = {
+        "TRADES": "TradeSummaryRequest",
+        "ORDERS": "OrderBookRequest",
+        "USER_ORDERS_TRADES": "OrderUpdateRequest",
+
+    }
+
+    WS_METHODS = {
+        "ORDERS_SNAPSHOT": "ob",
+        "ORDERS_UPDATE": "oi",
+        "TRADES_UPDATE": "ts",
+        "USER_BALANCE": "getTradingBalance",
+        "USER_ORDERS": "OrderResponse",
+        "USER_ORDERS_CANCEL": "OrderCancelResponse",
+    }
+
+    # Timeouts
+    MESSAGE_TIMEOUT = 30.0
+    PING_TIMEOUT = 10.0
+    API_CALL_TIMEOUT = 10.0
+    API_MAX_RETRIES = 4
+
+    # Intervals
+    # Only used when nothing is received from WS
+    SHORT_POLL_INTERVAL = 5.0
+    # CoinZoom poll interval can't be too long since we don't get balances via websockets
+    LONG_POLL_INTERVAL = 8.0
```

Comment:
Instead of using the `LONG_POLL_INTERVAL` to determine when the bot will update its balance, would it be better to simply just request for the balance every time there is an order update? 

@CoinAlpha/qa-team Were you able to capture user balance not updating?

Reply:
@CrimsonJacket we could do, but since this would be a REST request should I implement cachetools or something so as not to spam requests?

Reply:
Yes, during the initial start-up, took around 6 sec for the available balance to update.

Reply:
I do not think user data via the private websocket channel will be received that often. But I understand the concern with rate limits.

If it is a concern, maybe including a api_request throttler like in kraken would be good.

Reply:
@RC-13 How about between order proposals? Say after orders have been cancelled? There should be an edge case where, orders get cancelled but the available balance does not get updated, resulting in a "insufficient" balance being reported.

Reply:
@CrimsonJacket Right, sorry I forgot about this. We don't have a warning but it does experience inconsistency with order creation due to delay with updating of balance (similar behavior on other connectors). It's likely to be seen when running multiple orders and balance is not sufficient to cover all the levels.

Reply:
@CrimsonJacket I will make these changes later tonight!